treee111/wahooMapsCreator
# [FEATURE] restructure logging using logging-object, correct unittests & unittest-files

## This PR…

- changes all `print` outputs to log using the "global" logging object
- closes #59

## Considerations and implementations

In addition, these things are also within this PR:
- update / concrete conda .yml environments & renaming
- correction of unittests
- comitting of Windows unittest-files
- consistent usage of #/+/! as first letter in each logging output
- code refactoring to be consistent

## How to test

1. run wahooMapsCreator using CLI
   - check logging output

## Pull Request Checklist
    + Especially the [Pull-Requests](https://github.com/treee111/wahooMapsCreator/blob/develop/.github/CONTRIBUTING.md#Pull-Requests) section
